### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by jorgelainfiesta** [simplabs/breethe-client#98]
I used [aXe](https://www.axe-core.org/) Chrome extension to check accessibility issues on our app. The audit by this tool reported 8 issues regarding missing aria attributes, content landmarks and color contrasts. 

I'm planning to address these known issues now.

For the future, would it be worth it to incorporate an accessibility audit tool as part of our test suite? 

I'm not sure how much we're planning to extend the app in the future so I'm not sure if we should do it. Perhaps it's enough to run an audit tool manually an address the resulting issues.

If we want to incorporate an audit tool, there's several we could use. I've found we could use `aXe` with `mocca` or Google Chrome's `accessibility-developer-tools` with `puppeteer`. There's several interesting options we can go with.

**Comment by jorgelainfiesta** [comment]
Automated tools have troubles measuring the color contrasts of our app because of the fog effect background. However, I checked the color contrasts manually using an eyedropper over a screenshot and discovered that most of our blue shades don't have enough contrast to meet the recommendations. 

For now, I'll send a PR that only addresses the markup issues that we were facing plus some extra recommended practices.

**Comment by jorgelainfiesta** [comment]
This is my proposal for the color changes. I also took the liberty of making some minor style changes. I'll design the other screen and fog if you think this is the right direction @marcoow.
![artboard 1](https://user-images.githubusercontent.com/4451393/42593997-2150f7e4-850b-11e8-89ba-33ef0c0fd02d.png)

**Comment by marcoow** [comment]
I'd like to have a designer look at some parts of the app again tbh 😬. Let's postpone this for now and concentrate on the Sentry issues etc.
